adapters/lockerdome/lockerdome.go

```diff
@@ -0,0 +1,89 @@
+package lockerdome
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+)
+
+const unexpectedStatusCodeMessage = "Unexpected status code: %d. Run with request.debug = 1 for more info"
+
+// Implements Bidder interface.
+type LockerDomeAdapter struct {
+	endpoint string
+}
+
+// MakeRequests makes the HTTP requests which should be made to fetch bids [from the bidder, in this case, LockerDome]
+func (adapter *LockerDomeAdapter) MakeRequests(openRTBRequest *openrtb.BidRequest, extraReqInfo *adapters.ExtraRequestInfo) (requestsToBidder []*adapters.RequestData, errs []error) {
+
```

Comment:
Are we sure we are ready to marshall and return our request without validating the contents of our `openRTBRequest *openrtb.BidRequest`? Other adapters make sure to validate the elements of the `openRTBRequest.Imp[]` field first. Are we sure we are not sending
an empty or all-invalid-element `openRTBRequest.Imp[]` slice in our requestToBidder? Summarizing, what if:
  - `openRTBRequest.Imp[]` is empty or a null pointer? Is this request still worth posting?
  - All of `openRTBRequest.Imp[]`s elements are of media types other than `Banner`? According to your `static/bidder-info/lockerdome.yaml` file, `banner` is the only media type you support.
  - All of `openRTBRequest.Imp[]`s elements come with an empty `Ext` field? Would this request be useful?
  - At least one element in `openRTBRequest.Imp[]` comes with a non empty `Ext` field, are we sure we don't want to check whether or not it can be correctly unmarshalled without errors?

Reply:
```
// MakeRequests makes the HTTP requests which should be made to fetch bids [from the bidder, in this case, LockerDome]
func (adapter *LockerDomeAdapter) MakeRequests(openRTBRequest *openrtb.BidRequest, extraReqInfo *adapters.ExtraRequestInfo) (requestsToBidder []*adapters.RequestData, errs []error) {

 +  // Validation usually takes place here

	openRTBRequestJSON, err := json.Marshal(openRTBRequest)
	if err != nil {
		errs = append(errs, err)
		return nil, errs
	}

	headers := http.Header{}
	headers.Add("Content-Type", "application/json;charset=utf-8")
	headers.Add("Accept", "application/json")
	headers.Add("x-openrtb-version", "2.5")

	requestToBidder := &adapters.RequestData{
		Method:  "POST",
		Uri:     adapter.endpoint,
		Body:    openRTBRequestJSON,
		Headers: headers,
	}

	requestsToBidder = append(requestsToBidder, requestToBidder)

	return requestsToBidder, errs
}
```

Reply:
Thanks, I've added validations for the cases you mentioned. I'm seeing 96.4% coverage now.